.github/workflows/terratest-more-clusters.yaml

```diff
@@ -70,6 +70,14 @@ jobs:
           echo "Cluster 3 (cz):"
           kubectl get no -owide --context=k3d-test-gslb3
 
-      - name: Terratest
+      - name: Run Terratest
         continue-on-error: true
-        run: make terratest
+        run: mkdir -p ${{ github.workspace }}/tmp/terratest && make terratest | tee ${{ github.workspace }}/tmp/terratest/all.log
+
+      - name: Terratest log parser
+        run: terratest_log_parser --testlog ${{ github.workspace }}/tmp/terratest/all.log --outputdir ${{ github.workspace }}/tmp/terratest
+
+      - uses: actions/upload-artifact@v2
+        with:
+          name: terratest-logs
+          path: ${{ github.workspace }}/tmp/terratest        
```

Comment:
https://thoughtbot.com/blog/no-newline-at-end-of-file#history-lesson please keep the newline at the end of file and remove whitespace characters